src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.scss

```diff
@@ -0,0 +1,25 @@
+@import 'variables';
+
+:host {
+    background-color: white;
+    width: 100%;
+
+    ::ng-deep {
+        .container-selector {
+            height: auto;
+        }
+
+        .container-selector__list {
+            height: 100%;
```

Comment:
Should we do this for the grid-boxes?

Reply:
no, cuz grid boxes are limited in height and if you start adding a lot of containers then it will break